fabric-enchantment-api-v1/src/main/java/net/fabricmc/fabric/api/enchantment/v1/EnchantmentEvents.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.enchantment.v1;
+
+import java.util.function.BiFunction;
+
+import net.minecraft.enchantment.Enchantment;
+import net.minecraft.item.ItemStack;
+
+import net.fabricmc.fabric.api.event.Event;
+import net.fabricmc.fabric.api.event.EventFactory;
+import net.fabricmc.fabric.api.util.TriState;
+
+/**
+ * Events to intercept the vanilla enchantment logic.
+ *
+ * @author Vaerian (vaeriann@gmail.com or @Vaerian on GitHub).
+ *
+ * <p>Please contact the author, Vaerian, at the email or GitHub profile listed above
```

Comment:
Don't think author is visible in generated javadoc unless you turn `-author` flag on. I suggest moving the contacts out of the `@author` block tag.

Reply:
I'm not fond of the `@author` annotation, and I would like it not to be used, when it comes to Fabric API.

Reply:
Yeah when something is in fabapi it's usually maintained by contributors alike than one specific person. That's a good point.